cmake/ngraph.cmake

```diff
@@ -48,5 +48,11 @@ ExternalProject_Add(
 
 set(NGRAPH_INCLUDE_DIR ${NGRAPH_INSTALL_PREFIX}/include)
 set(NGRAPH_LIB_DIR ${NGRAPH_INSTALL_PREFIX}/lib)
+if(EXISTS "/etc/centos-release")
+    EXECUTE_PROCESS(COMMAND uname -m COMMAND tr -d '\n' OUTPUT_VARIABLE ARCHITECTURE)
+    if(${ARCHITECTURE} STREQUAL "x86_64")
+        set(NGRAPH_LIB_DIR ${NGRAPH_INSTALL_PREFIX}/lib64)
+    endif()
+endif()
```

Comment:
CMake has a [`find_library`](https://cmake.org/cmake/help/v3.0/command/find_library.html) function.  Would it make sense to use that instead?  I think it would be a little more idiomatic, and might help us avoid having CentOS-specific logic.